lib/simplecov/version.rb

```diff
@@ -1,5 +1,5 @@
 module SimpleCov
-  version = "0.14.1"
+  version = "0.14.1".dup
 
   def version.to_a
     split(".").map(&:to_i)
```

Comment:
Yeah, I'm not a fan either. Whatever you'd prefer for https://github.com/colszowka/simplecov-html/pull/56, I'll do the same here as well.